ma/config/magento.py

```diff
@@ -11,3 +11,40 @@
 
 _LOGGER.debug("CONFIG: S=[%s] U=[%s] P=[%s] H=[%s] P=(%d)",
               SCHEME, USERNAME, PASSWORD, HOSTNAME, PORT)
+
+# Hash used to execute remote re-indexing and cache refresh.
+UTILITY_HASH = os.environ.get('MAGENTO_UTILITY_HASH', None)
```

Comment:
I renamed it to MAGENTO_SECURITY_TOKEN. 
I don't want to throw an exception if the variable is not set. Caching/Indexing should not prevent the process to run. I modified to raise the exception upon class instantiation, if missing. at least the main functionality would run by then. 
I initially added the exception to both methods because I envisioned using MaintenanceApi for other things that might not require the Security Token.